src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -318,6 +317,19 @@ export default App.extend({
         },
       });
     }
+
+    const shouldShowOwnerToggle = this.getState().id === 'shared-by' && this.canViewAssignedActions;
+
+    if (shouldShowOwnerToggle) {
+      const ownerToggleView = listTitleView.showChildView('ownerToggle', new NoOwnerToggleView({
+        model: new Backbone.Model(filters),
```

Comment:
For some reason, this was throwing errors if I didn't set this as a `new Backbone.Model()`. Not sure why...